include/unifex/create.hpp

```diff
@@ -36,7 +36,7 @@ struct _op {
     template (typename... Ts)
       (requires receiver_of<Receiver, Ts...>)
     void set_value(Ts&&... ts) noexcept(is_nothrow_receiver_of_v<Receiver, Ts...>) {
-      unifex::set_value((Receiver&&) rec_, (Ts&&) ts...);
+      unifex::set_value((Receiver&&) rec_, (ValueTypes) ts...);
```

Comment:
> First, I think there's a chance that the conversion from Ts&& to ValueTypes might throw, right? If so, the current noexcept declaration is incomplete.

Thanks, good catch.

 > Second, I intend to change set_value be noexcept

Done. Makes sense. Once `set_value` is mandated to be `noexcept`, we can do the same cosntexpr conditional to avoid the try/catch if the conversion is nonthrowing, but I have not made that change here.

 > Finally, we've been standardizing on an unstated coding style: use std::move() for moves and static_cast<Dest>(value) to replace std::forward, and don't use C-style casts.

I made one step towards this within this PR, although I think I have at least one more place to update within my changes. Thanks for clarifying the standard within this project. I will note, clang-tidy has been adding CppCoreGuidelines checks for correct move/forward/universal parameter usage (e.g., [F.18](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-consume)) and I think the checks work best (or in some cases, only work) when `move` and `forward` are used rather than casts.

Reply:
>  I will note, clang-tidy [fixes this]

Huh.  We should take advantage. I've filed issue #548.